tests/syntax/test_errors.py

```diff
@@ -307,17 +308,16 @@ def runFile(path):
 
 @pytest.mark.parametrize('bug', (
     # BUGGY CODE                    ERROR CAUGHT DURING:
-    'x = 3 << 2',                   # token translation
     '4 = 2',                        # Python parsing
     '3 relative to',                # Python parsing
-    'Point at x y',                 # Python parsing (with offset past end of original line)
+    'new Point at x y',                 # Python parsing (with offset past end of original line)
```

Comment:
Please fix the alignment of these comments, and update the categories: this one should be "Scenic parsing" now, and the parenthesis no longer makes sense.

Reply:
Fixed in 4a046ec9ee90b764d367a17d55e7c4f20cedcdb5